RULES.md

```diff
@@ -2260,29 +2260,39 @@ A route's color and `route_text_color` should be contrasting.
+| ----------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ["N"/"Judah"](https://www.sfmta.com/getting-around/transit/routes-stops/n-judah)                | [Muni San Fransisco](https://storage.googleapis.com/storage/v1/b/mdb-latest/o/us-california-san-francisco-municipal-transportation-agency-sfmta-gtfs-50.zip?alt=media) |
+| ["6"/"ML King Jr Blvd"](https://trimet.org/schedules/r006.htm)                                  | [Trimet Portland Streetcar](http://developer.trimet.org/schedule/gtfs.zip)                                                                                             |
+| ["55"/"Boulevard Saint Laurent"](https://www.stm.info/en/info/networks/bus/local/line-55-north) | [STM Montreal](https://storage.googleapis.com/storage/v1/b/mdb-latest/o/ca-quebec-societe-de-transport-de-montreal-gtfs-1221.zip?alt=media)                            |
+| ["1"/"Rangiora/Cashmere"](https://www.metroinfo.co.nz/timetables/1-rangiora-cashmere/)          | [Metro Christchurch](https://storage.googleapis.com/storage/v1/b/mdb-latest/o/nz-christchurch-christchurch-metro-gtfs-1313.zip?alt=media)                              |
+
+Bad examples:
+| `route_short_name`/`route_long_name` |
+|-------------------------------------------|
+| "604"/"604"                               |
+| "14"/"Route 14"                           |
+| "2"/"Route 2: Bellows Falls In-Town"      |
 
-| `route_id` 	| `route_short_name` 	| `route_long_name` 	|
-|------------	|--------------------	|-------------------	|
-| route1     	| L1                 	| L1                	|
 
 #### References
-* [routes.txt specification](http://gtfs.org/reference/static/#routestxt)
+* [routes.txt Best Practices](https://gtfs.org/schedule/best-practices/#routestxt)
 <details>
 
 #### Notice fields description
-| Field name     	| Description                             	  | Type   	|
-|----------------	|-------------------------------------------	|--------	|
-| `routeId`        	| The id of the faulty record.            	| String  |
-| `csvRowNumber`   	| The row number of the faulty record.    	| Long 	  |
-| `routeShortName` 	| The faulty record's `route_short_name`. 	| String 	|
-| `routeLongName`  	| The faulty record's `route_long_name`.  	| String 	|
+| Field name     	| Description                               | Type   	|
+|-----------------	|---------------------------------------- |--------	|
+| `routeId`       	| The id of the faulty record.           	| String 	|
+| `csvRowNumber`  	| The row number of the faulty record.   	| Long   	|
+| `routeShortName` 	| The `short_name` of the faulty record.  | String 	|
```

Comment:
Could you please replace `short_name` with `route_short_name` to be consistent with the spec?